src/pages/publication/references_tab/components/InstitutionSearch.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC, useCallback, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+
+import { AutoSearch } from '../../../../components/AutoSearch';
+import { searchFailure } from '../../../../redux/actions/searchActions';
+import useDebounce from '../../../../utils/hooks/useDebounce';
+import { Institution } from '../../../../types/institution.types';
+import { queryInstitution } from '../../../../api/institutionApi';
+import i18n from './../../../../translations/i18n';
+
+interface InstitutionSearchProps {
+  clearSearchField: boolean;
+  dataTestId: string;
+  label: string;
+  setValueFunction: (value: any) => void;
+  placeholder?: string;
+}
+
+const InstitutionSearch: FC<InstitutionSearchProps> = ({
+  clearSearchField,
+  dataTestId,
+  label,
+  setValueFunction,
+  placeholder,
+}) => {
+  const [searchResults, setSearchResults] = useState<Institution[]>([]);
+  const [searchTerm, setSearchTerm] = useState('');
+  const [searching, setSearching] = useState(false);
+
+  const debouncedSearchTerm = useDebounce(searchTerm);
+  const dispatch = useDispatch();
+  const { t } = useTranslation('feedback');
```

Comment:
dette søket vil muligens få samme problematikk som på ProjectSearch (at search ikke blir triggered hvis man skriver inn den samme searchTerm'en to ganger)?

Reply:
ja, det samme skjer her. Sjekk ut ProjectSearch.tsx for å se hvordan det er gjort der (hele search-funksjonen er debouncet)